service/src/test/resources/core-case-data-store-api/populated-case-details.json

```diff
@@ -5,6 +5,7 @@
   "case_type_id": "Shared_Storage_DRAFTType",
   "security_classification": "PUBLIC",
   "case_data": {
+    "caseLocalAuthority": "example",
```

Comment:
FYI Sam updates to this file are likely to break DocumentGeneratorServiceTests. Update to submitted-form-pdf-content.txt to reflect your changes.

Reply:
nope, tests not broken, does this test do what you think it does?

Reply:
Just tested, you're right. Matching is more forgiving than I remembered it being 👍